clipboard-apis/async-write-html-read-html.https.html

```diff
@@ -19,9 +19,19 @@
 // but when we are comparing for equality the spaces make a difference.
 function reformatHtml(html) {
   const parser = new DOMParser();
-  const htmlString =
-        parser.parseFromString(html, 'text/html').documentElement.innerHTML;
-  const reformattedString = htmlString.replace(/\>\s*\</g, '> <');
+  const parsedHtml = parser.parseFromString(html, 'text/html')
+
+  // More canonical form
+  // Remove the head - not visible to users
+  const headElement = parsedHtml.head;
+  if (headElement) { headElement.remove(); }
+  // Remove the eventual style attributes
+  const elementsWithStyleAttribute = parsedHtml.querySelectorAll('[style]');
+  elementsWithStyleAttribute.forEach((element) => { element.removeAttribute('style'); });
+  const htmlString = parsedHtml.documentElement.innerHTML;
```

Comment:
Maybe `document.createRange().createContextualFragment(html)` to skip all of these?

Reply:
another solution would be to not necessarily check that we get the same string at all and remove this part of the test. We know the type needs to be html, but there is nothing which defines what html it should be.
The mime type is already checked.